File: cfg-keys/src/main/java/net/dblsaiko/qcommon/cfg/keys/ext/KeyBindingExt.java
package net.dblsaiko.qcommon.cfg.keys.ext;

import net.minecraft.client.options.KeyBinding;

public interface KeyBindingExt {

    void incrTimesPressed();

    static KeyBindingExt from(KeyBinding self) {
        return (KeyBindingExt) self;
    }

}

File: src/main/kotlin/therealfarfetchd/commoncfg/client/binds/KeyBindingExt.kt
package therealfarfetchd.commoncfg.client.binds

import net.minecraft.client.options.KeyBinding

interface KeyBindingExt {

  fun setPressed(value: Boolean)

  fun incrTimesPressed()

}

fun KeyBinding.setPressed(value: Boolean) =
  (this as KeyBindingExt).setPressed(value)

fun KeyBinding.incrTimesPressed() =
  (this as KeyBindingExt).incrTimesPressed()